teos/extended_appointment.py

```diff
@@ -36,11 +56,20 @@ def from_dict(cls, appointment_data):
 
         appointment = Appointment.from_dict(appointment_data)
         user_id = appointment_data.get("user_id")
+        user_signature = appointment_data.get("user_signature")
+        start_block = appointment_data.get("start_block")
 
-        if not user_id:
-            raise ValueError("Wrong appointment data, user_id is missing")
+        if any(v is None for v in [user_id, user_signature, start_block]):
```

Comment:
Could be more compact with:
```python
if None in [user_id, user_signature, start_block]:
```

Although here perhaps it would be even better to check each field individually to give more precise error messages?

Reply:
I was thinking there was a good reason for this, but turns out there is none. This comes from a refactor of `all(v is not None for v in ...` which I actually overlooked at the time ([guess who suggested it](https://github.com/talaia-labs/python-teos/pull/57#discussion_r343459366) :P).

I'll change this and the equivalent in `Appointment` in a separate PR.

Reply:
Lol. Well, my reviews improve jointly with my python :P